src/schemas/viber/bodies/cards/viberCard.json

```diff
@@ -0,0 +1,59 @@
+{
+  "title": "viberCard",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": ["type", "elements"],
+  "properties": {
+    "type": {
+      "type": "string",
+      "enum": ["vertical"],
+      "default": "vertical",
+      "readonly": true
+    },
+    "tag": {
+      "type": "string",
+      "default": "generic",
+      "readonly": true
+    },
+    "alt": {
+      "type": "string",
+      "description": "This property has no effect in Viber, but is included in the schema to ensure the compatibility of Line Cards."
+    },
+    "elements": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": ["type", "elements"],
+        "properties": {
+          "type": {
+            "type": "string",
+            "enum": ["vertical"],
+            "default": "vertical",
+            "readonly": true
+          },
+          "elements": {
+            "type": "array",
+            "maxItems": 7,
+            "items": {
+              "anyOf": [
+                {
+                  "$ref": "viberImage.json"
+                },
+                {
+                  "$ref": "viberTitle.json"
+                },
+                {
+                  "$ref": "viberSubtitle.json"
+                },
+                {
+                  "$ref": "viberButton.json"
+                }
```

Comment:
@joshuajott This implies a lot of different card configurations. For example a card which would contain 7 subtitle elements would be possible according to this schema. Are you sure **all of the possibilities** will be actually working?

If not, consider defining valid card configurations like it has been done with [Line](https://github.com/LivePersonInc/rich-content-validator/tree/master/src/schemas/line/bodies/cards).